docs/addsegment-guide.md

```diff
@@ -17,14 +17,14 @@
 
 # Heterogeneous format segments in carbondata
 
-###Background
+### Background
 In the industry, many users already adopted to data with different formats like ORC, Parquet, JSON, CSV etc.,  
 If users want to migrate to Carbondata for better performance or for better features then there is no direct way. 
 All the existing data needs to be converted to Carbondata to migrate.  
 This solution works out if the existing data is less, what if the existing data is more?   
 Heterogeneous format segments aims to solve this problem by avoiding data conversion.
 
-###Add segment with path and format
+### Add segment with path and format
 Users can add the existing data as a segment to the carbon table provided the schema of the data
  and the carbon table should be the same. 
 
```

Comment:
Looks like syntax is wrong. Should be
Alter table table_name add segment options (‘path’= 'hdfs://usr/oldtable,'format'='parquet')

Reply:
done